crates/admin/src/cluster_controller/logs_controller.rs

```diff
@@ -37,18 +39,19 @@ use restate_types::logs::metadata::{
 };
 use restate_types::logs::{LogId, Lsn, TailState};
 use restate_types::metadata_store::keys::BIFROST_CONFIG_KEY;
-use restate_types::nodes_config::Role;
 use restate_types::partition_table::PartitionTable;
 use restate_types::replicated_loglet::{
-    NodeSet, ReplicatedLogletId, ReplicatedLogletParams, ReplicationProperty,
+    ReplicatedLogletId, ReplicatedLogletParams, ReplicationProperty,
 };
 use restate_types::retries::{RetryIter, RetryPolicy};
 use restate_types::{logs, GenerationalNodeId, NodeId, PlainNodeId, Version, Versioned};
 
 use crate::cluster_controller::observed_cluster_state::ObservedClusterState;
 use crate::cluster_controller::scheduler;
+use nodeset_selection::{NodeSelectionError, NodeSetSelectionStrategy, NodeSetSelector};
+use restate_types::nodes_config::NodesConfiguration;
 
-type Result<T, E = LogsControllerError> = std::result::Result<T, E>;
+pub type Result<T, E = LogsControllerError> = std::result::Result<T, E>;
```

Comment:
Why making this type public?

Reply:
It was necessary previously, but not anymore; ty!